tools/check-source.sh

```diff
@@ -83,6 +83,10 @@ fgrep -n '\^' $texfiles |
 grep -n 'U+' $texfiles |
     fail 'use \\unicode or \\ucode or \\uname instead' || failed=1
 
+# Discourage double-wrapping \tcode{\exposid{data_}}
+grep -n '\\tcode{\\exposid{[a-zA-Z0-9_]*}}' $texfiles |
+    fail 'double-wrapped \\exposid in \\tcode' || failed=1
```

Comment:
#6406 also has a commit for `\exposconcept` in `\tcode`.
There might be yet other macros similarly affected.

Reply:
Ok. Let's take this first and then add more.